owncloudApp/src/main/java/com/owncloud/android/presentation/ui/sharing/fragments/ShareFileFragment.kt

```diff
@@ -406,12 +445,12 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
      *********************************************** PRIVATE SHARES ***********************************************
      **************************************************************************************************************/
 
-    private fun updatePrivateShares(privateShares: List<OCShareEntity>) {
+    private fun updatePrivateShares(privateShares: List<OCShare>) {
         // Get Users and Groups
         this.privateShares = ArrayList(privateShares.filter {
```

Comment:
A cat dies every time I see ArrayList instead of List. I don't understand why you convert it here.

Reply:
Done

![tenor](https://user-images.githubusercontent.com/15261363/68931367-a8638180-0790-11ea-9070-e2a7b711cb0e.gif)